src/Shared/UnitTests/TypeLoader_Dependencies_Tests.cs

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
+using System;
+using System.IO;
+using Microsoft.Build.Shared;
+using Microsoft.Build.SharedUtilities;
+using Xunit;
+
+namespace Microsoft.Build.UnitTests
+{
+    public class TypeLoader_Dependencies_Tests
```

Comment:
Rather than adding this new test file and new test projects, could you instead modify PortableTask to add a dependency to it? That should test task dependencies without the need to add extra projects or tests.